Documentation/ReleaseProcess.md

```diff
@@ -0,0 +1,112 @@
+## Release Process ##
+
+### Step 1: Creating a release branch ###
+
+The branch structure in Aztec iOS is based on the GIT Flow methodology.
+
+These are the main branches involved in the release process:
+```
+master: Kept up to date with the latest release.
+develop: Bleeding edge branch with all of the latest changes.
+release/x.y.z: Version branch, which we will use during the release process.
+```
+The first step in the process is creating the version branch.
+
+The version branch is what allows work to continue in develop while also having a frozen snapshot of what we’re releasing.
+
+Using the command line, the process is pretty straightforward:
+
+```
+git checkout develop
+git pull
+git checkout -b release/x.y.z
+git push origin -u release/x.y.z
+```
+
+Make sure to update the field `s.version` in both of these files to the `x.y.z` number
+```
+WordPress-Aztec-iOS.podspec
+WordPress-Editor-iOS.podespec
+```
+
+Open a PR on github where you target the `master` branch with the `release/x.y.z` branch and call it `Release x.y.z`
+
+
+### Step 2: Testing the Integration ###
+
+Before going any further, It's normally good practice to test the Aztec integration into WordPress-iOS project, to make sure I won’t have to do the release process twice.
```

Comment:
* Lowercase "i" in "It's"
* With the change from "I" to "it's" we now also need to update "make sure I won't...". Could be something like "make sure we won't...".